source/js/components/petition/donation-modal.jsx

```diff
@@ -74,18 +85,21 @@ class DonationModal extends Component {
       this.donateURL = `?form=donate`;
     }
     return (
-      <div className="modal-content" role="dialog">
+      <div
+        className={classNames(`modal-content`, MODAL_CONTENT_CLASS)}
```

Comment:
Similarly, we need to keep `modal-content` class here as we are still relying on Bootstrap's stylings for `.modal-content`